docs/user-guide/liveness/http-liveness.yaml

```diff
@@ -13,6 +13,8 @@ spec:
       httpGet:
         path: /healthz
         port: 8080
+        httpHeaders:
+          - ['X-Custom-Header', 'Awesome']
```

Comment:
It's at the bottom of my YAML bible, [Complete idiot's introduction to YAML](https://github.com/Animosity/CraftIRC/wiki/Complete-idiot's-introduction-to-yaml). Fixed.